packages/crayons-core/src/components/datepicker/datepicker.tsx

```diff
@@ -204,6 +206,18 @@ export class Datepicker {
    * Label displayed on the interface, for the component.
    */
   @Prop() label = '';
+  /**
+   * Whether the time-picker should be shown in the date-picker.
+   */
+  @Prop() time = false;
+  /**
+   * The props for the time picker.
+   */
+  @Prop() timeProps = {};
+  /**
+   * The format of time picker .
+   */
+  @Prop() timeFormat: 'HH:mm' | 'hh:mm a' = 'hh:mm a';
```

Comment:
Now, with this attribute being introduced, can we consider renaming existing `displayFormat` to something like `dateDisplayFormat` to reflect that it points to date alone ?

Reply:
Can we just use the `displayFormat` and allow the user to pass the format with time or just date format alone - displayFormat for timepicker - `yyyy/MM/dd HH:mm`
displayFormat for datepicker -  `yyyy/MM/dd`

Reply:
If we use a single prop for both time and date format the used can use any pattern to enter the date time format and the format need to be split to get the time format and passed to `fw-timepicker`. The drawback with that approach is the end user can use any pattern to set the format like `d-MM-yy HH:mm` or  `d-MM-yy - HH:mm a` or even the worst case scenario `HH:mm - yyyy/MM/dd` in cased like these it difficult to get the time format from the display format. Hence i decided to for different props for date and time format as it easy to validate the too.

Reply:
Can you please check if `date-fns` supports parsing date-time formats . If that's case, using just a single format will help. `date-fns` util can return the appropriate date and time format.

Reply:
Also pls check if i18n support works with Timepicker as well. For datepicker the format changes based on the lang. Similarly I would advise we change the Timepicker's format as appropriate.

Reply:
Added Locale support to Time picker. Now the timepickers format will be based on the locale if the format/timeFormat
 is not provide